tests/tests.py

```diff
@@ -55,14 +95,14 @@ def getTestsList(collect_only: bool, test_name_filter: str, gdbinit_path: str) -
         exit(0)
 
     # Extract the test names from the output using regex
-    pattern = re.compile(r"tests/.*::.*")
+    pattern = re.compile(rf"{test_dir_path}.*::.*")
```

Comment:
The output of pytests_collect.py that we care about puts the name of each test on a new line starting with:

`Test: `

Perhaps we change this regex to search for something that starts with this?

Reply:
if that makes sense then sure